server/website/website/settings/constants.py

```diff
@@ -24,3 +24,6 @@
 
 # The background tasks only process workloads containing this minimum amount of results
 MIN_WORKLOAD_RESULTS_COUNT = 5
+
+# The views used for metrics pruning
+VIEWS_FOR_PRUNING = ['dba_hist_osstat', 'dba_hist_sysstat', 'dba_hist_system_event']
```

Comment:
This option should be a dict of database_type -> views_for_pruning. If a database_type is not in the dict or its value is empty/None then that should indicate to use all views. This should fix the unit test errors.

(a) Add 'dba_workload_replays' to the list too please.
(b) Are you skipping the metrics in the 'dba_hist_sys_time_model' view because there are so many of them?

```
from website.types import DBMSType
...

VIEWS_FOR_PRUNING = {
    DBMSType.ORACLE: ['dba_hist_osstat', 'dba_hist_sysstat', 'dba_hist_system_event', 'dba_workload_replays'],
}
```

Reply:
I add dba_workload_replays and dba_hist_sys_time_model to the list.